_education/gi-bill/higher-learning.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
this page has a markup error somewhere with a stray div at the bottom. (it's a pre-existing issue, so could be fixed in another PR)